<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CGetReRegionsWidget</class>
 <widget class="QWidget" name="CGetReRegionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>5</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_blackBang">
      <attribute name="title">
       <string>电池区域</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="resFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>4</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="layout_battery">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_dark">
      <attribute name="title">
       <string>中间极耳区域</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="layout_midregion">
         <property name="spacing">
          <number>0</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_white">
      <attribute name="title">
       <string>胶带粘贴区域</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="layout_blueTape"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_Hsv">
      <attribute name="title">
       <string>两侧胶带连接</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="layout_dblTape"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_19">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string> 电池排列方向</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comb_batDire">
          <item>
           <property name="text">
            <string>上下</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>左右</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="grpbx_battery">
        <property name="title">
         <string>电池区域</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>网格宽</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spin_batteryGridW">
             <property name="maximum">
              <number>999</number>
             </property>
             <property name="value">
              <number>50</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>网格长</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spin_batteryGridH">
             <property name="maximum">
              <number>999</number>
             </property>
             <property name="value">
              <number>50</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_18">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>阈值下限</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spin_batteryMinThre">
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="value">
              <number>240</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>腐蚀量</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spin_batterryEro">
             <property name="maximum">
              <number>999</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_17">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>区阈数</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spin_batterySlctNum">
             <property name="maximum">
              <number>10</number>
             </property>
             <property name="value">
              <number>2</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="btn_battery">
           <property name="text">
            <string> 测试</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="grpbx_mid">
        <property name="title">
         <string>中间区域</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_20">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>区域膨胀</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spin_midDila">
             <property name="maximum">
              <number>9999</number>
             </property>
             <property name="value">
              <number>1400</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>阈值上限</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spin_midThre">
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="value">
              <number>140</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_21">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>腐蚀值</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spin_midero">
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="value">
              <number>10</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="btn_mid">
           <property name="text">
            <string> 测试</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="grpbx_tapes">
        <property name="title">
         <string>蓝色胶带</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>高度膨胀</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spin_tapeHDila">
             <property name="maximum">
              <number>999</number>
             </property>
             <property name="value">
              <number>500</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_25">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>宽度膨胀</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spin_tapeWDila">
             <property name="maximum">
              <number>999</number>
             </property>
             <property name="value">
              <number>150</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_24">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>胶带数量</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spin_tapeNum">
             <property name="maximum">
              <number>10</number>
             </property>
             <property name="value">
              <number>4</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="btn_tapeNum">
           <property name="text">
            <string> 测试</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="grpbx_dbltapes">
        <property name="title">
         <string>胶带连接区域</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_26">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>极耳方向</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cmbx_dblDirect">
             <item>
              <property name="text">
               <string>上下</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>左右</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>腐蚀量</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spin_dblEro">
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="value">
              <number>10</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="btn_dblTape">
           <property name="text">
            <string> 测试</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_test">
        <property name="text">
         <string>检测</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_savePam">
        <property name="text">
         <string>参数保存</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spin_batteryGridH</sender>
   <signal>valueChanged(int)</signal>
   <receiver>btn_battery</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1027</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>1028</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_batterryEro</sender>
   <signal>valueChanged(int)</signal>
   <receiver>btn_battery</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1027</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>1028</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_midThre</sender>
   <signal>valueChanged(int)</signal>
   <receiver>btn_mid</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1027</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>1028</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_tapeHDila</sender>
   <signal>valueChanged(int)</signal>
   <receiver>btn_tapeNum</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1027</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>1028</x>
     <y>436</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_tapeWDila</sender>
   <signal>valueChanged(int)</signal>
   <receiver>btn_tapeNum</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1027</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>1028</x>
     <y>436</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_tapeNum</sender>
   <signal>valueChanged(int)</signal>
   <receiver>btn_tapeNum</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1027</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel">
     <x>1028</x>
     <y>436</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_dblEro</sender>
   <signal>valueChanged(int)</signal>
   <receiver>btn_dblTape</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1027</x>
     <y>499</y>
    </hint>
    <hint type="destinationlabel">
     <x>1028</x>
     <y>523</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_batteryGridW</sender>
   <signal>valueChanged(int)</signal>
   <receiver>btn_battery</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>988</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>987</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_batteryMinThre</sender>
   <signal>valueChanged(int)</signal>
   <receiver>btn_battery</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>970</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>964</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_batterySlctNum</sender>
   <signal>valueChanged(int)</signal>
   <receiver>btn_battery</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>978</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>980</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_midDila</sender>
   <signal>valueChanged(int)</signal>
   <receiver>btn_mid</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>958</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>915</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_midero</sender>
   <signal>valueChanged(int)</signal>
   <receiver>btn_mid</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>973</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>931</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
